raiden/api/python.py

```diff
@@ -493,13 +495,18 @@ def channel_batch_close(
             token_address=token_address,
         )
 
+        greenlets: typing.List[Greenlet] = list()
         for channel_state in channels_to_close:
             channel_close = ActionChannelClose(
                 token_network_identifier=token_network_identifier,
                 channel_identifier=channel_state.identifier,
             )
 
-            self.raiden.handle_state_change(channel_close)
+            greenlets.extend(
```

Comment:
So this is the only place where you don't use `handle_and_track_state_change()` and the reason is that you want to be sure the handling of the events is finished (thus the join) before you continue further down, correct?

Reply:
Oh I see.

Reply:
Otherwise they would be raised  in the service and not handled properly by the rest API to generate proper error responses I suppose.